Makefile

```diff
@@ -19,8 +22,22 @@ create-application-user:
 	CREATE_SECRET=secret-application.yml.tpl \
 	sh ./db-ops/create-db-user.sh || echo
 
+create-application-dev-user:
+	for u in ${CREATE_DB_DEV_USERS}; do \
+	REGION=${region} \
+	SEED=${randomSeed} \
+	PROJECT_NAME=${PROJECT_NAME} \
+	ENVIRONMENT=${ENVIRONMENT} \
+	NAMESPACE=${PROJECT_NAME} \
```

Comment:
We probably shouldn't be creating these secrets in the real staging application namespace. How about somewhere else?
Do we even need these to be secrets? I'm assuming you're making secrets so you can copy them into the dev's namespace later, but how about just using aws secrets manager? If they are all in a namespace they will be lost if something happens to that namespace.

Reply:
see above comment